test/extended/authentication/front_proxy.go

```diff
@@ -33,6 +33,14 @@ var _ = g.Describe("[sig-auth][Feature:Authentication] ", func() {
 			if *controlPlaneTopology == configv1.ExternalTopologyMode {
 				e2eskipper.Skipf("External clusters do not have an aggregator-client secret in the cluster. Because the control plane lives outside the cluster, the aggregator-client secret is not needed in the cluster.")
 			}
+
+			// Temporary workaround until https://issues.redhat.com/browse/OCPBUGS-17431 is fixed.
+			isMicroShift, err := exutil.IsMicroShiftCluster(oc.AdminKubeClient())
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if isMicroShift {
+				e2eskipper.Skipf("MicroShift clusters do not have an aggregator-client secret in the cluster")
```

Comment:
```suggestion
				e2eskipper.Skipf("MicroShift team promised to create this test separately because the aggregator client cert is accessed differently there. If this text is still here, you should ask them to provide details about where their test lives. ")
```

Reply:
/cc @pacevedom @dhellmann are we good adding this skip message?

Reply:
Its ok with me.